OutOfSchool/OutOfSchool.WebApi/Extensions/ElasticsearchExtension.cs

```diff
@@ -31,7 +31,8 @@ public static void AddElasticsearch(this IServiceCollection services, ElasticCon
         var settings = new ConnectionSettings(pool)
             .DefaultIndex(config.WorkshopIndexName)
             .BasicAuthentication(config.User, config.Password)
-            .EnableApiVersioningHeader();
+            .EnableApiVersioningHeader()
+            .ServerCertificateValidationCallback((a, b, c, d) => true);
```

Comment:
I wonder how this change is related to the ratings. Could you provide more details?

Reply:
I`m sorry. I think that`s left since I couldn`t run elastic. I`ll remove it now